client/src/ui/components/resources/EntityResourceTable.tsx

```
@@ -27,24 +23,22 @@ export const EntityResourceTable = ({ entityId }: { entityId: ID | undefined })
         maxStorehouseCapacityKg={maxStorehouseCapacityKg}
         tick={tick}
       />
     ));
-
-    return (
+    return entityId !== undefined && entityId !== null ? (
       <div key={tier}>
         <div className="grid grid-cols-1 flex-wrap">{resources}</div>
       </div>
+    ) : (
+      <div>No Entity Selected</div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)